changed he to they (#3401)
javascript-tutorial/en.javascript.info: 6-data-storage/02-localstorage/1-form-autosave/task.md

Create a `textarea` field that "autosaves" its value on every change.

> So, if the user accidentally closes the page, and opens it again, they'll find his unfinished input at place.

Reviewer: ```suggestion
So, if the users accidentally close the page, then open it again, they'll find his unfinished input at place.
```

if we're going plural, we may need to switch to users also

Reviewer 2: ```suggestion
So, if the users accidentally close the page, then open it again, they'll find their unfinished input at place.
```
his can also be changed to their

Reviewer 3: [singular they](https://en.wikipedia.org/wiki/Singular_they) works fine IMO, no need for `users`
